### PR TITLE
Django 1.10 compatibility

### PR DESCRIPTION
**Pull request by blag**
This allows tests to run with Django 1.10. The dependency on django-extensions was updated to 1.7.0 to avoid the `SubfieldBase` issues from the Django version bump.

### CODE REVIEW

File: dnsaas/api/urls.py [hunk 1]
```diff
@@ -1,4 +1,17 @@
-from django.conf.urls import include, patterns, url
+from django import VERSION
```
Comment (ar4s) [inline dnsaas/api/urls.py]: please move this bunch of code to module (e.g. ``compat.py``) to avoid breaking DRY principle

**Comment by blag** [comment]
Derp; you're totally right! Currently away from code but I'll fix this tonight. Thanks for the review!

**Comment by blag** [comment]
@ar4s I fixed the issue you pointed out. Is there a way to get CircleCI to run tests on this?

**Comment by mkurek** [comment]
this should be merged at the same time as #171 (to not break our dnsaas with newer Django)

**Review by ghost — APPROVED** [review]

**Review by szok — APPROVED** [review]

**Comment by mkurek** [comment]
@blag since #171 was unfortunately merged first, is this PR still relevant (I think that it's part of #171)

**Comment by blag** [comment]
@mkurek Yeah, this PR was a part of that. I was intending to make them independent, but I don't think that worked out as I wanted.

But whatever, the end result is the same.

**Pull request closed without merging**